test_data/download_data.sh

```diff
@@ -18,7 +20,7 @@ tar -xzvf refdata-cellranger-hg19-3.0.0.tar.gz
 # Download common variants
 wget --load-cookies /tmp/cookies.txt "https://docs.google.com/uc?export=download&confirm=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate 'https://docs.google.com/uc?export=download&id=1lw4T6d7uXsm9dt39ZtEwpuB2VTY3wK1y' -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')&id=1lw4T6d7uXsm9dt39ZtEwpuB2VTY3wK1y" -O common_variants_hg19.vcf && rm -rf /tmp/cookies.txt
 wget https://master.dl.sourceforge.net/project/cellsnp/SNPlist/genome1K.phase3.SNP_AF5e2.chr1toX.hg19.vcf.gz
-gunzip genome1K.phase3.SNP_AF5e2.chr1toX.hg19.vcf.gz
+#gunzip genome1K.phase3.SNP_AF5e2.chr1toX.hg19.vcf.gz
```

Comment:
Why?

Reply:
Because cellsnp can use the compressed file directly, dont need to unzip it. will delete the comment